python/mxnet/log.py

```diff
@@ -81,7 +81,6 @@ def getLogger(name=None, filename=None, filemode=None, level=WARNING):
     """Gets a customized logger.
 
     .. note:: `getLogger` is deprecated. Use `get_logger` instead.
-
```

Comment:
Are you sure about this? I tend to see sphinx rst errors when you don't leave an extra line break after using a `..`

Reply:
Thanks! I fixed it.